plugins/scanners/scanoss/src/test/assets/exclusionTest/ArchiveUtils.kt

```diff
@@ -0,0 +1,245 @@
+/*
```

Comment:
If I understand it right, you copied a source file from ORT to use as test data ?

I am not too sure with this approach as it will lead to "fake" matches when somebody search for code references in the project.

Reply:
I used the same approach that was already established in the [filesToScan folder](https://github.com/oss-review-toolkit/ort/tree/main/plugins%2Fscanners%2Fscanoss%2Fsrc%2Ftest%2Fassets%2FfilesToScan) as a reference for my implementation.

If needed, I can certainly modify the test files in the `filesToScan` folder and `exclusionTest` folder to avoid potential fake matches during code reference searches.

Reply:
I created a separate [commit](https://github.com/oss-review-toolkit/ort/pull/10004/commits/c546cdded681836170a673a5958b20a5a99fc303) for this change since it addresses a different concern from the main feature work. Would you prefer to have this change in the same commit where the files were originally added, or is it okay to keep it as a separate commit?

Reply:
Thanks, let's leave it like that then.